extension/updateswh.js

```diff
@@ -53,6 +53,7 @@ function testupdateforge(url, forgespecs) {
 	var projecturl = forgespecs.projecturl;
 	var userproject = forgespecs.userproject;
 	var forgeapiurl = forgespecs.forgeapiurl;
+	var forgeapidata = fogespecs.forgeapidata;
```

Comment:
var forgeapidata = fo**r**gespecs.forgeapidata;

Reply:
Thanks @fgaz for this! May you split the PR in two parts, so we can merge right away support for forges that require a POST message, that may be useful for other use cases?